cypress/e2e/pages/transactions/[txid].spec.ts

```diff
@@ -280,10 +272,11 @@ context("/transactions/[txid] on mobile", () => {
             .should("have.text", "OP_DEFI_TX_SET_ORACLE_DATA");
           cy.get("span").eq(2).should("have.text", "0.00000000 DFI");
         });
-
       cy.get("div")
         .next()
+        .first()
         .within(() => {
+          // some pure magic here
```

Comment:
I think this a suppose to be for a self reference note? If yes then maybe we can remove this comment

Reply:
removed